internal/pkg/manifest/validate.go

```diff
@@ -810,7 +874,16 @@ func (d *DeadLetterQueue) Validate() error {
 }
 
 // Validate returns nil if OverrideRule is configured correctly.
-func (*OverrideRule) Validate() error {
+func (r *OverrideRule) Validate() error {
+	if r == nil {
+		return nil
+	}
+	for _, s := range invalidTaskDefOverridePathRegexp {
+		re := regexp.MustCompile(fmt.Sprintf(`^%s$`, s))
+		if re.MatchString(r.Path) {
```

Comment:
I'd love for us to go through this file at some point and remove basic regex's from the codebase

Reply:
Yeah agree. As for this PR it is safe I think since we are mostly strictly moving things around.